SmellDetector/SmellModel/Issue.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace SmellDetector.SmellModel
+{
+    public class Issue
+    {
+        public SmellType IssueType { get; set; }
+
+        public string Problem { get; set; }
+
+        public Issue()
+        {
+            IssueType = SmellType.WITHOUT_BAD_SMELL;
+            Problem = "Problem is: ";
+        }
+    }
```

Comment:
## Comment

With this code, I try to reply to what is the smell and where it is.

![image](https://user-images.githubusercontent.com/45834270/97647859-91767780-1a53-11eb-9999-9ae4401fa2c6.png)

## Example which detects where is the smell

Something like this is the output: **'Problem is Long parameter list in private void AddSomeFeatureExample()'**

I see it is not a generic way and needs more improvement, but for now, I hope is good enough?

![image](https://user-images.githubusercontent.com/45834270/97648009-f762ff00-1a53-11eb-9003-2b24f5fd7e0d.png)

Reply:
It is enough to have the actual ID (your pair.key) - let the **View** worry about which text will be displayed. Here we are focused on the data - the issues related to a code element.